### ISSUE TITLE
Add --format=env option to config get command

### ISSUE THREAD

**Issue by abbatis** [wp-cli/config-command#98]
## Feature Request

**Describe the solution you'd like**

Currently the `--format` options are `var_export`, `json`, and `yaml`. It would be nice to have an `env` option here as well, which would print something like this:

```
DB_NAME=wp_site
DB_USER=wp_user
DB_PASSWORD=123123
```

**Comment by schlessera** [comment]
Yes, that would be quite handy for `dotenv` usage. Are you interested in providing a PR for this?

**Comment by abbatis** [comment]
@schlessera ☝️ There u go. I'll keep an eye out for any comments.

**Comment by schlessera** [comment]
@abattis The `dotenv` format is very specific to config files, I don't think it makes sense to add it as a generic option to the framework that is available to all commands.

You should provide it within the config command only, by handling it directly, while forwarding all other formats to the generic formatter.

You can see an example of how this is done for the contextual formats `ids` & `count` here: https://github.com/wp-cli/entity-command/blob/master/src/Post_Command.php#L616-L634

**Comment by schlessera** [comment]
Are you still up for this, @imadphp ?

**Comment by imadphp** [comment]
Yes, definitely. Will update it soon. @schlessera

**Comment by imadphp** [comment]
@schlessera Thank you for your patience and for pointing out the example in wp-cli/entity-command. I hope this is better. Not sure if it's 100% according to your code standards, so let me know if there's anything I can improve.

**Comment by swissspidy** [comment]
Looks like this was fixed by #102 a long time ago.
